spec/support/omniauth_macros.rb

```diff
@@ -74,4 +78,66 @@ def mock_auth_existing_user_admin
                               }
                             )
   end
+
+  def mock_auth_accounts
```

Comment:
With some modifications we can remove up to one of the previous mock accounts. We can't replace more than one because of the unique username each of these mock accounts has.
If we removed the usernames then we could replace more of the previous mock accounts, but then the mock accounts defined in mock_auth_accounts would not be identical to the ones in development.rb